src/App.js

```diff
@@ -101,6 +94,21 @@ class App extends React.Component {
     const buttons = document.getElementsByClassName("gsc-search-button gsc-search-button-v2")
     buttons[0].click();
   }
+
+  autoTranslate(input) {
+    this.allowTabs()
+    
+    // updates state with what user typed
+    this.setState({
+      input: input.target.value
+    })
+
+    if (this.timeout) clearTimeout(this.timeout);
+    this.timeout = setTimeout(() => {
+      // translates if 300 ms passes after last keystroke
+      this.postRequest()
+    }, 300);
```

Comment:
Nit: 300ms seems like it might be a little short, especially when typing fancy code characters, but this would be super easy to change in the future. My main concern is that showing an error to the user as they're typing might be intimidating.

Reply:
+1 if for the error case you show a big error currently.
Consider some options to alleviate this:
- Don't show the error unless you have some confidence that the user has finished typing i.e. show it if the user has not typed for O(seconds)
- Incorporate better affordances in the UI for errors. i.e. leaving error cases untranslated and highlighting them red.